sdlf-ddk-lightweight/data_lake/pipelines/sdlf_pipeline_stack.py

```diff
@@ -0,0 +1,236 @@
+                "name": f"{team}-{pipeline}-stage-a",
+                "type": "octagon_pipeline",
+                "description": f"{resource_prefix} data lake light transform",
+                "id": f"{team}-{pipeline}-stage-a"
+            },
+            environment_id=self._environment_id,
+            config=SDLFLightTransformConfig(
+                team=team,
+                pipeline=pipeline,
+                raw_bucket = self._foundations_stage.raw_bucket,
+                raw_bucket_key = self._foundations_stage.raw_bucket_key,
+                stage_bucket = self._foundations_stage.stage_bucket, 
+                stage_bucket_key = self._foundations_stage.stage_bucket_key,
+                routing_lambda = self._foundations_stage.routing_function, 
+                data_lake_lib = self._foundations_stage.data_lake_library,
+                register_provider = self._foundations_stage.register_provider,
+                wrangler_layer = self._wrangler_layer
+            ),
+        )
+        
+        data_lake_heavy_transform = SDLFHeavyTransform(
+            self,
+            id=f"{pipeline_id}-stage-b",
+            name=f"{team}-{pipeline}-stage-b",
+            prefix = resource_prefix,
+            description=f"{resource_prefix} data lake heavy transform",
+            props={
+                "version": 1,
+                "status": "ACTIVE",
+                "name": f"{team}-{pipeline}-stage-b",
+                "type": "octagon_pipeline",
+                "description": f"{resource_prefix} data lake heavy transform",
+                "id": f"{team}-{pipeline}-stage-b"
+            },
+            environment_id=self._environment_id,
+            config=SDLFHeavyTransformConfig(
+                team=team,
+                pipeline=pipeline,
+                stage_bucket = self._foundations_stage.stage_bucket, 
+                stage_bucket_key = self._foundations_stage.stage_bucket_key,
+                data_lake_lib = self._foundations_stage.data_lake_library,
+                register_provider = self._foundations_stage.register_provider,
+                wrangler_layer = self._wrangler_layer,
+                database_crawler_name = f"{resource_prefix}-{team}-{dataset}-post-stage-crawler"
+            ),
+        )
+        self.routing_b = data_lake_heavy_transform.routing_lambda
+    
+        self._data_lake_pipeline: DataPipeline = (
+            DataPipeline(
+                self, 
+                id=logical_data_pipeline_id,
+                name=f"{logical_data_pipeline_id}-pipeline",
+                description=f"{self._resource_prefix} data lake pipeline", 
+            )
+            .add_stage(data_lake_s3_event_capture_stage)
+            .add_stage(data_lake_light_transform)
+            .add_stage(data_lake_heavy_transform, skip_rule=True)
+        )
+        return data_lake_heavy_transform
+
+    def _create_custom_pipeline(self):
+        return
+
+    def _create_wrangler_layer(self):
```

Comment:
Are we assuming this example only works in `us-east-1`?

Reply:
It should be region agnostic, but i am not sure which regions support aws-data-wrangler lambda layer. i can parameterize region

Reply:
We support all commercial regions now for wrangler layers.